src/botPage/bot/TradeEngine/Ticks.js

```diff
@@ -51,21 +52,26 @@ export default Engine =>
                 this.$scope.ticksService.request({ symbol: this.symbol }).then(ticks => {
                     let lastTick = raw ? getLast(ticks) : getLast(ticks).quote;
                     if (toString && !raw) {
+                        if (!toString) {
+                            lastTick = parseFloat(lastTick);
```

Comment:
I think we can revert this part of code to the previous one. No need to change it ..